### PR TITLE
add minimal-deploy-topology.md

### PR DESCRIPTION
**Pull request by superlzs0476**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: minimal-deployment-topology.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: 常规拓扑架构
+category: how-to
+---
+
+# 部署需求
+
+通过 `tidb` 用户做集群管理，使用默认 `22` 端口，部署目录为 `/tidb-deploy`，数据目录为 `/tidb-data`。
```
Comment (kissmydb) [inline minimal-deployment-topology.md:8]:
```suggestion
部署一个常规集群，采用最小拓扑，包括 3 * tidb-server、3 * tikv-server、3 * pd-server。
```
端口和目录感觉放这有点不合群，放在配置那里如何

File: minimal-deployment-topology.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+title: 常规拓扑架构
+category: how-to
+---
+
+# 部署需求
```
Comment (kissmydb) [inline minimal-deployment-topology.md:6]:
```suggestion
# 部署目标
```

File: minimal-deployment-topology.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+# 拓扑图
+
+待添加
+
+# 配置文件模版 topology.yaml
+
+[简单最小拓扑配置](/simple-mini.yaml)
+
+[详细最小拓扑配置](/complex-mini.yaml)
```
Comment (ran-huang) [inline minimal-deployment-topology.md:27]: These links are not valid.

File: minimal-deployment-topology.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+---
+title: 常规拓扑架构
```
Comment (ran-huang) [inline minimal-deployment-topology.md:2]: The title for this doc is `最小部署拓扑结构`. Please confirm whether `常规` and `最小` mean the same thing or not.

File: minimal-deployment-topology.md (hunk 3 above)
Comment (kissmydb) [reply minimal-deployment-topology.md:27]: pr is not merge

Comment (kissmydb) [reply minimal-deployment-topology.md:30]: 现在有了

File: minimal-deployment-topology.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
+---
+title: 最小拓扑架构
+summary: 介绍 TiDB 集群的最小拓扑。
+category: how-to
+---
+
+# 常规拓扑架构
```
Comment (ran-huang) [inline minimal-deployment-topology.md:7]: 一级标题与 metadata 的 title 须一致。
```suggestion
# 最小拓扑架构
```

**Pull request merged**